pkg/functions/client.go

```diff
@@ -826,10 +826,7 @@ func (c *Client) Run(ctx context.Context, f Function) (job *Job, err error) {
 	}()
 
 	if !f.Initialized() {
-		// TODO: this needs a test.
-		err = fmt.Errorf("the given path '%v' does not contain an initialized "+
-			"function.  Please create one at this path in order to run", f.Root)
-		return nil, err
+		return nil, fmt.Errorf("Can not run an uninitialized function")
```

Comment:
```suggestion
		return nil, fmt.Errorf("can not run an uninitialized function")
```
Nit: I think convention is to begin error messages with lower-case